designsafe/apps/auth/views.py

```diff
@@ -64,15 +67,13 @@ def tapis_oauth(request):
 
 def launch_setup_checks(user):
     """Perform any onboarding checks or non-onboarding steps that may spawn celery tasks"""
-    systems_to_configure = [
-        {"system_id": settings.AGAVE_STORAGE_SYSTEM, "path": user.username},
-        {"system_id": settings.AGAVE_WORKING_SYSTEM, "path": user.username},
-    ]
+    systems_to_configure = get_systems_to_configure(user.username)
     client = user.tapis_oauth.client
 
     for system in systems_to_configure:
         system_id = system["system_id"]
         path = system["path"]
+        create_path = system["create_path"]
         try:
             client.files.listFiles(systemId=system_id, path=path)
             logger.debug(
```

Comment:
from slack discussions
- [ ] move as much as possible to workers (i.e. 3 systems listings)
- [ ] double check if any other delay exists coming from during login